c++/src/kj/compat/http.h

```diff
@@ -234,9 +234,20 @@ class HttpHeaderTable {
   kj::StringPtr idToString(HttpHeaderId id) const;
   // Get the canonical string name for the given ID.
 
+  bool isValid() const;
```

Comment:
Sorry to bikeshed but something like `isReady()` might be a clearer name...

Reply:
That's a good point, I'll switch it.